internal/bus/topics.go

```diff
@@ -6,19 +6,21 @@
 package bus
 
 const (
-	AddInstancesTopic          = "add-instances"
-	UpdatedInstancesTopic      = "updated-instances"
-	DeletedInstancesTopic      = "deleted-instances"
-	ResourceUpdateTopic        = "resource-update"
-	NginxConfigUpdateTopic     = "nginx-config-update"
-	InstanceHealthTopic        = "instance-health"
-	ConfigUploadRequestTopic   = "config-upload-request"
-	DataPlaneResponseTopic     = "data-plane-response"
-	ConnectionCreatedTopic     = "connection-created"
-	ConfigApplyRequestTopic    = "config-apply-request"
-	WriteConfigSuccessfulTopic = "write-config-successful"
-	ConfigApplySuccessfulTopic = "config-apply-successful"
-	ConfigApplyFailedTopic     = "config-apply-failed"
-	RollbackCompleteTopic      = "rollback-complete"
-	RollbackWriteTopic         = "rollback-write"
+	AddInstancesTopic           = "add-instances"
+	UpdatedInstancesTopic       = "updated-instances"
+	DeletedInstancesTopic       = "deleted-instances"
+	ResourceUpdateTopic         = "resource-update"
+	NginxConfigUpdateTopic      = "nginx-config-update"
+	InstanceHealthTopic         = "instance-health"
+	ConfigUploadRequestTopic    = "config-upload-request"
+	DataPlaneResponseTopic      = "data-plane-response"
+	ConnectionCreatedTopic      = "connection-created"
+	ConfigApplyRequestTopic     = "config-apply-request"
+	WriteConfigSuccessfulTopic  = "write-config-successful"
+	ConfigApplySuccessfulTopic  = "config-apply-successful"
+	ConfigApplyFailedTopic      = "config-apply-failed"
+	RollbackCompleteTopic       = "rollback-complete"
+	RollbackWriteTopic          = "rollback-write"
+	DataplaneHealthTopic        = "dataplane-health"
+	DataplaneHealthProcessTopic = "dataplane-health-process"
```

Comment:
```suggestion
	DataplaneHealthRequestTopic   = "dataplane-health-request"
	DataplaneHealthResponseTopic = "dataplane-health-response"
```

Maybe request & response might be better names for the topics?